packages/tabs/src/utils/getSelectedIndex/getSelectedIndex.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param selected
+ * @param tabTitleElements
+ * @returns Returns the corresponding index for a selected string
+ */
+export const getSelectedIndex = (
+  selected: number | string,
+  tabTitleElements: Array<HTMLElement>,
+) => {
+  if (typeof selected === 'number') return selected;
+
+  return tabTitleElements.findIndex(
+    element => element.dataset.text === selected,
+  );
```

Comment:
What happens if there are multiple tabs with the same name? 
I'm fine with the default `.findIndex` behavior (resolve to the first one), but we should add a unit test for this case.

Reply:
i'll add a test.